backend/tests/v5/unit/models/teamspaces.test.js

```diff
@@ -427,6 +427,17 @@ const testRemoveUserFromAdminPrivileges = () => {
 	});
 };
 
+const testGetAllTeamspacesWithActiveLicenses = () => {
+	describe('Get all teamspaces with active licenses', () => {
+		test('Should perform a query to find all teamspaces with active subscriptions', async () => {
+			const expectedRes = [generateRandomString()];
+			const fn = jest.spyOn(db, 'find').mockResolvedValueOnce(expectedRes);
+			await expect(Teamspace.getAllTeamspacesWithActiveLicenses()).resolves.toEqual(expectedRes);
+			expect(fn).toHaveBeenCalledTimes(1);
```

Comment:
we could check the find params ?

Reply:
the query is too complex, the copy and paste isn't worth it imo. it just makes the test harder to read

can add it if you really think it's worth while

Reply:
sure its ok I guess